src/main/java/dog/pawbook/model/managedentity/dog/Sex.java

```diff
@@ -0,0 +1,35 @@
+package dog.pawbook.model.managedentity.dog;
+
+import static java.util.Objects.requireNonNull;
+
+public class Sex {
+
+    private final String value;
```

Comment:
does it mean dogs can have the entire spectrum of sexuality now?

Reply:
Just male and female

Reply:
Hmm, then do we want a validation regex just like the date?